### PR TITLE
Bugfix asset not found fix

### PR DESCRIPTION
**Pull request by marcelkuipers**
I think this will fix the AssetNotFoundException for everyone.
It seems like the asset manager uses /target/classes/ as base directory, which is also where all other resources are located. I moved the xml files to the asset folder and now Nifty allows me to address the files directly.

### CODE REVIEW

**Comment by LMiljak** [comment]
Very strange that moving it in the asset folder before, but it does work now. Glad it's fixed though.

File: ContextDesktop/src/main/java/com/github/migi_1/Context/screens/MainMenu.java [hunk 1]
```diff
@@ -38,13 +38,13 @@
      * Creates the start screen.
      */
      private void createStartScreen() {
-         nifty.fromXml("com/github/migi_1/Context/resources/main_screen.xml", "start", mainMenuController);
+         nifty.fromXml("Screens/main_screen.xml", "start", mainMenuController);
```
Comment (MrCrime) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/screens/MainMenu.java]: This should have been fixed before, so now is the right time:
Make a private static final attribute of this.

File: ContextDesktop/src/main/java/com/github/migi_1/Context/screens/MainMenu.java [hunk 2]
```diff
@@ -38,13 +38,13 @@
...
      }
      /**
       * Creates the host screen.
       */
      private void createHostScreen() {
-         nifty.addXml("com/github/migi_1/Context/resources/host_screen.xml");
+         nifty.addXml("Screens/host_screen.xml");
```
Comment (MrCrime) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/screens/MainMenu.java]: Same for this.

**Comment by MrCrime** [comment]
Also, what I think should have been done before, but now is the right time:
One method uses fromXml method and the other addXml, but both methods are described as "Creates the ... screen." What's the difference?

**Comment by LMiljak** [comment]
I'm merging it right now then. (Seems a bit pointless to wait for a second review for such a small PR)

**Pull request merged**